### ISSUE TITLE
luaposix33 fails to compile because of missing term.h

### ISSUE THREAD

**Issue by mk-f** [mkschreder/mks-openwrt-feed#6]
```
'/home/fabian/playground/openwrt/build_dir/target-arm_arm926ej-s_musl-1.1.14_eabi/luaposix-release-v33.3.1'
  CC       ext/posix/posix.lo
  In file included from ext/posix/ctype.c:22:0,
                   from ext/posix/posix.c:13:
           ./ext/include/_helpers.c:38:18: fatal error: term.h: No
           such file or directory
```

The strange thing is, that it seems like the #include term.h in /ext/include/_helpers.c is not needed at all, removing the line "fixes" the build.

**Comment by nunojpg** [comment]
+1

**Comment by zooboa** [comment]
+1

**Comment by roccozhang** [comment]
+1

**Comment by Johnson-Fan** [comment]
+1

**Comment by cfcnightmare** [comment]
Any advance here?
Deleting line and then make gives a hell of other errors and build failed

**Comment by cfcnightmare** [comment]
Fixed!
Just change in Luaposix **Makefile** from:

PKG_NAME:=luaposix33
PKG_VERSION:=v33.3.1
PKG_RELEASE:=1

PKG_SOURCE:=release-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://github.com/luaposix/luaposix/archive/
PKG_MD5SUM:=1d07112a5582b539930066d7c834643f

to:

PKG_NAME:=luaposix33
PKG_VERSION:=v33.4.0
PKG_RELEASE:=1

PKG_SOURCE:=release-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://github.com/luaposix/luaposix/archive/

A deleted MD5 sum!!!  works just norm :)

Close please!

**Comment by mkschreder** [comment]
Fixed since 2.16.08.
